web/docs/tutorials/todo-app/03-listing-tasks.md

```diff
@@ -18,13 +19,34 @@ To list tasks, we will need two things:
 1. A Wasp query that fetches all the tasks from the database.
 2. React logic that calls our query and displays its results.
 
-## Wasp query
+## Defining the Query
 
 Let's implement `getTasks` [query](language/features.md#query).
 It consists of a declaration in Wasp and implementation in JS (in `src/server/` directory).
 
 ### Wasp declaration
 Add the following code to `main.wasp`:
+
+<Tabs groupId="js-ts">
+<TabItem value="js" label="JavaScript">
+
+```wasp title="main.wasp"
+// ...
+
+query getTasks {
+  // We specify that JS implementation of the query (which is an async JS function)
+  // can be found in `src/server/queries.js` as the named export `getTasks`.
+  // Use '@server' to reference files inside the src/server folder.
+  fn: import { getTasks } from "@server/queries.js",
+  // We tell Wasp that this query is doing something with entity `Task`. With that, Wasp will
+  // automatically refresh the results of this query when tasks change.
+  entities: [Task]
+}
+```
+
+</TabItem>
+<TabItem value="ts" label="TypeScript">
+
```

Comment:
Most Wasp files don't require a toggle, but I duplicated them anyway because:
1. Something about those extensions might change in the near future.
2. I often show additional content/explanations below the code snippet for TypeScript.
3. It helps reassure the user by effectively telling them "hey, it really is the same for both languages"